pkgs/development/coq-modules/mathcomp-analysis/default.nix

```diff
@@ -8,6 +8,7 @@ let mca = mkCoqDerivation {
   pname = "analysis";
   owner = "math-comp";
 
+  release."0.3.10".sha256 = "sha256-FBH2c8QRibq5Ycw/ieB8mZl0fDiPrYdIzZ6W/A3pIhI=";
```

Comment:
If the old version is not deleted, the commit message's "title" should be `coqPackages.mathcomp-analysis: init at 0.3.10`, with some explanation such as "including the new version along the older one".

Reply:
According to `git log`, the convention for `coqPackages` is what I have in the PR. It is understood that for this package set, old versions are not discarded.